src/reaper_osara.cpp

```diff
@@ -868,6 +868,12 @@ void postToggleAllTracksFxBypass(int command) {
 		translate("all tracks FX active"));
 }
 
+void postToggleLastFocusedFxDeltaSolo(int command) {
+	outputMessage(GetToggleCommandState(command) ?
+		translate("delta solo on") :
+		translate("delta solo off"));
```

Comment:
Given the length of the words "delta solo", what do you think about "delta on"/"delta off"? Alternatively, "enabled delta solo"/"disabled delta solo"? My aim here is to make sure the users get the important information (the on/off state) as early as possible.

Reply:
Or "soloed delta" and "unsoloed delta", but that seems weird.

Reply:
I think Enabled/disabled delta solo is ok. The word solo seems pretty relevant in this context.